### DISCUSSION TITLE
Glitch in custom addres fields on page checkout

### DISCUSSION THREAD

**Discussion by wdmarketLV** [breezefront/community#111] [1. Help] [answered]
When I add custom address fields and also I have one step checkot, I get erro in checkout when place order. 
Can you please check does your extension work? 

![image](https://user-images.githubusercontent.com/79901550/233110692-75cfea2c-f1ff-4b6a-8aa1-c01fce82761f.png)

**Comment by vovayatsyuk** [comment, marked as answer]
Sorry, but you have to contact swissup/firecheckout support (depending on where you purchased the module). This forum is only for breeze-specific questions.

**Reply by wdmarketLV** [reply]
OK thanks
Just support usually sends here for questions
